modules/theme/src/theme-core/definitions/globals/product.less

```diff
@@ -435,6 +462,8 @@ body {
                 Theme Icon
 *****************************************/
 
+@iconMiniVeryPadding: 15px;
```

Comment:
Shall we add this to a variable? Check other places too.

Reply:
Theme icon styles will be moved to a separate definition i.e `definitions/elements/theme-icon.less`. `@iconMiniVeryPadding` will be a variable in that module. So moving `@iconMiniVeryPadding` to `site.variables` is not required IMO.